_data-prepper/pipelines/configuration/processors/aggregate.md

```diff
@@ -20,8 +20,33 @@ group_duration | No | String | The amount of time that a group should exist befo
 
 <!---## Configuration
 
-Content will be added to this section.
+Content will be added to this section.--->
 
 ## Metrics
 
-Content will be added to this section.--->
\ No newline at end of file
+Apart from common metrics in the [Abstract processor](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-api/src/main/java/org/opensearch/dataprepper/model/processor/AbstractProcessor.java), the aggregate processor introduces the following custom metrics:
+
+**Counter**
+
+* `actionHandleEventsOut`: The number of events that have been returned from the `handleEvent` call to the [action](#action) configured
```

Comment:
@graytaylor0 Is there another page this should link to? We don't currently have an Action section in this document.

Reply:
Yes. The full link we should be linking to here is https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/aggregate-processor#action